gulpfile.babel.js

```diff
@@ -9,7 +9,7 @@ const plugins = loadPlugins({
   lazy: false,
 });
 
-import babelRegister from 'babel/register';
+import babelRegister from 'babel-core/register';
```

Comment:
I can't find any docs on it, but it looks like babel is the cli and babel-core will just be the compiler parts.